### PR TITLE
Add support for MSON Refract

### PR DESCRIPTION
**Pull request by kylef**
This pull request adds support for the latest version of the API Blueprint AST which removes MSON AST. This pull request does not remove MSON AST support and leaves backwards compatibility.

I would like to go down this path for now, and later clean up all the login with a proper Refract library in Swift.

### CODE REVIEW

File: Hyperdrive/HyperBlueprint.swift [hunk 1]
```diff
@@ -452,3 +410,36 @@ func +<K,V>(lhs:Dictionary<K,V>, rhs:Dictionary<K,V>) -> Dictionary<K,V> {
...
+        return false
+      }
+      .flatMap { $0["value"] as? Element }
+      .first
+  }
+}
+
+private func selectElementValue(element: Element) -> String? {
+  return element["element"] as? String
+}
```
Comment (w-vi) [inline Hyperdrive/HyperBlueprint.swift]: *nitpick*: missing new line?

**Comment by w-vi** [comment]
Not a swifter but it LGTM as least on the abstract level. :+1:

**Comment by pksunkara** [comment]
I still don't have access to merge this.

**Comment by w-vi** [comment]
me neither

**Pull request merged**